server/src/controllers/parcels/parcelDetails.ts

```diff
@@ -2,7 +2,8 @@ import parcelDetailsDB from '../../database/queries/parcelsQueries';
 
 const parcelDetails = async (req: any, res: any) => {
   const { id } = req.params;
-  const { rows } = await parcelDetailsDB(id);
+  const { userId } = req;
+  const { rows } = await parcelDetailsDB(id, userId);
   if (rows.length === 0) {
```

Comment:
it is better if we destruct rowCount instead of rows, then check rows.length.

Reply:
done